### ISSUE TITLE
Update to cf-release 213

### ISSUE THREAD

**Issue by bsideup** [yudai/cf_nise_installer#133]
Hi!

Could you please update scripts to the latest version of CF (213 at this moment)?

Subtasks:
- [ ] Update cf-release submodule to 213

Will try to do it by myself and send pull request, but not sure that I have enough knowledge for it :)

Thanks!

**Comment by yudai** [comment]
Currently I cannot update this project any longer.The newer cf-release has

**Comment by rboykin** [comment]
Persistent disk requirement removed here:  https://github.com/cloudfoundry-incubator/etcd-release/commit/8b04ae5b68d252ac3f23001aafa88139e72f0fd6

**Comment by yudai** [comment]
Trying with v214. I'll report once I got result.

**Comment by yudai** [comment]
Updated!

**Comment by jabrena** [comment]
Hi Yudai, you say that exist a new release?

**Comment by yudai** [comment]
@jabrena Yes.

**Comment by jabrena** [comment]
Many thanks @yudai.

Next week, I will try it!

Any additional command to run:

``` shell

sudo apt-get install curl
bash < <(curl -s -k -B https://raw.githubusercontent.com/yudai/cf_nise_installer/${INSTALLER_BRANCH:-master}/scripts/bootstrap.sh)

```

**Comment by rboykin** [comment]
I tried it.  Deployed fine.  All components started with the exception of some consul problems which did not seem to hinder me.  But, also warden does not stay up.  I believe due to the use of the -w option on iptables in the commit here:  https://github.com/cloudfoundry/warden/commit/d667339af644776573c1c06b8cd5344e477d6caf  Out of curiosity I removed the -w options in the two files from the commit and then warden stays up.

I am running on Ubuntu 10.04 as recommended.  Is Ubuntu 14 supported as iptables there has support for -w option?
